src/main/java/org/prebid/server/auction/AuctionRequestFactory.java

```diff
@@ -231,11 +230,99 @@ private HttpRequestWrapper toHttpRequest(HookStageExecutionResult<EntrypointPayl
                 .build();
     }
 
+    private Future<BidRequest> parseBidRequest(HttpRequestWrapper httpRequest, List<String> errors) {
+        try {
+            final JsonNode bidRequestNode = bodyAsJsonNode(httpRequest.getBody());
+
+            final String referer = paramsExtractor.refererFrom(httpRequest);
+            ortbTypesResolver.normalizeBidRequest(bidRequestNode, errors, referer);
+
+            return Future.succeededFuture(jsonNodeAsBidRequest(bidRequestNode));
+        } catch (Exception e) {
+            return Future.failedFuture(e);
+        }
+    }
+
+    private Future<AuctionContext> extractAccountAndExecuteRawAuctionHooks(BidRequest bidRequest,
+                                                                           HttpRequestWrapper httpRequestWrapper,
+                                                                           HookExecutionContext hookExecutionContext,
+                                                                           long startTime,
+                                                                           List<String> errors) {
+        final Timeout timeout = timeout(bidRequest, startTime, timeoutResolver);
+        return accountFrom(bidRequest, timeout, httpRequestWrapper)
+                .compose(account -> executeRawAuctionRequestHooksAndUpdate(
+                        bidRequest,
+                        httpRequestWrapper,
+                        account,
+                        hookExecutionContext,
+                        errors,
+                        timeout));
+    }
+
+    private Future<AuctionContext> executeRawAuctionRequestHooksAndUpdate(BidRequest bidRequest,
```

Comment:
This method name is a bit misleading too - `Update` suffix is too generic and forces to look into the body to find out what it's doing.

Reply:
Remove this to not duplicate such a long name